server/migrations/20220203231624-fix-ion.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkUpdate('Chains', {
+        symbol: 'ION',
+        network: 'ion',
+        base: 'cosmos',
+        bech32_prefix: 'ion',
+      }, {
+        id: 'ion',
+      }, { transaction: t });
+
+      await queryInterface.bulkUpdate('ChainNodes', {
+        url: 'https://rpc-osmosis.keplr.app',
```

Comment:
We need to change this URL to https://rpc-osmosis.blockapsis.com (+ if it's hardcoded somewhere), and we also need to start providing alt_wallet_url as https://lcd-osmosis.blockapsis.com... I may make a separate push to fix this stuff.

Reply:
👍